admin/client/App/index.js

```diff
@@ -21,8 +21,6 @@ Keystone.User = listsByKey[Keystone.userList];
 const { adminPath } = Keystone;
 
 const doRender = () => {
-	const Routes = require('./Routes').default;
```

Comment:
actually, it might even work simply with

```js
import Routes from './Routes';
...
   module.hot.accept('./Routes', () =>  {Routes = require('./Routes')});
```

I don't think imports are `const`…
Oh and the render needs to run too.